tests/fast/ReportConflictingKeys.txt

```diff
@@ -0,0 +1,10 @@
+testTitle=ReportConflictingKeysTest
+    testName=ReportConflictingKeys
+    testDuration=20.0
+    nodeCount=10000
+    keyPrefix=RCK
+    keyBytes=64
+    readConflictRangeCountPerTx=10
+    writeConflictRangeCountPerTx=10
+    connectionFailuresDisableDuration=100000
+    buggify=off
```

Comment:
Why do we have to turn off buggify for this?

Reply:
that's an excellent question. @zjuLcg can we remove this line?

Reply:
I think it's intentional, see https://github.com/apple/foundationdb/pull/2257#discussion_r390035115. My guess is that it's not strictly necessary, but it improves coverage since we tr.commit() failing with say commit_unknown_result is not interesting here.

Reply:
I think it follows the same way as `ConflictRange` workload.

Reply:
What Andrew is saying makes sense - though even with buggify on I would expect most transactions would not fail with `commit_unknown_result`...

I wonder whether this causes some code-paths not getting tested well enough. I see that @etschannen added this to `ConflictRangeCheck.txt` ~2 years ago. Evan, can you share your opinion on this.